reana_job_controller/job_manager.py

```diff
@@ -103,13 +103,13 @@ def create_job_in_db(self, backend_job_id):
             cvmfs_mounts=self.cvmfs_mounts or '',
             shared_file_system=self.shared_file_system or False,
             docker_img=self.docker_img,
-            cmd=self.cmd,
+            cmd=json.dumps(self.cmd),
```

Comment:
If we do this, we should change REANA-DB models so `cmd` and `prettified_cmd` are JSON columns, what do you think? For now, it can stay like this, but once we later go and read this value from DB we will get a string instead of a Python object which is less convenient. I think it makes sense to store it like JSON  since for all computing backends we need to split between `command`/`executable`... and `args`/`arguments` etc.. 👍 

Do you mind doing it in a different commit? thanks!